#!/usr/bin/env python
PACKAGE = "tro"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("securityFactor",   double_t,   0, "securityFactor" , 1, 1, 10)
gen.add("separation",       double_t,   0, "separation"     , 4, 1, 20)
gen.add("dt",               double_t,   0, "dt"             , 0.1, 0.001, 1)

gen.add("axmaxAccel",       double_t,   0, "axmaxAccel"     , 7, 0.1, 50)
gen.add("axmaxDecel",       double_t,   0, "axmaxDecel"     , 7, 0.1, 50)
gen.add("aymax",            double_t,   0, "aymax"          , 7, 0.1, 50)

gen.add("savePath",         str_t,      0,  "savePath"      ,"/home/fetty/Escritorio/control_ws2021/src/control/tro/TROpy/")
gen.add("midlinePath",      str_t,      0,  "midlinePath"   ,"/home/fetty/Escritorio/control_ws2021/src/control/tro/TROm/Tracks/Germany2019.txt")


exit(gen.generate(PACKAGE, "tro", "dynamic"))